CHANGES.md

```diff
@@ -86,6 +86,15 @@ description indicates whether it is SOURCE-COMPATIBLE, BINARY-COMPATIBLE, or BRE
 
 #### Arm
 
+* Support SGIs (software generated interrupts) on platforms with GICv2 or GICv3 in non-SMP configurations. SGIs are
+  intended for signalling other cores when seL4 is used in a multi-kernel setup. On Arm, this is implemented with a new
+  capability `SGISignal`, which can be created from `IRQControl` capabilities with the new IRQ control invocation
+  `ARMIRQIssueSGISignal` for a specific IRQ and target core. The resulting `SGISignal` capability can be invoked like a
+  notification capability that supports only signal/send. SGIs can be received by IRQ notification objects on the target
+  core like other IRQs.
+
+  See also [RFC-17](https://sel4.github.io/rfcs/implemented/0170-multikernel-ipi-api.html)
```

Comment:
Will people get confused by IPI in the target link? Probably doesn't matter

Reply:
Not much we can do about there if we don't want to launch into an explanation of IPI/SGI etc. The RFC text does explain, though. So the confusion should only last until then (of course nobody ever clicks on the link and reads things).